modules/doc/content/newsletter/index.md

```diff
@@ -5,8 +5,13 @@ major changes to the project monthly highlights will be produced. These highligh
 monthly to the [MOOSE users](contact_us.md) and [MOOSE announcement](contact_us.md) lists as well as
 provided below.
 
+## 2022
+
+- [January, 2022](2022_01.md)
+
```

Comment:
Please remove. The index link to the page is only added at the beginning of the next month.